docs/project/plugins.sbt

```diff
@@ -11,5 +11,5 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-less"   % "1.1.2")
 addSbtPlugin("com.typesafe.sbt" % "sbt-uglify" % "2.0.0")
 
 addSbtPlugin("de.heikoseeberger" % "sbt-header"         % "5.2.0")
-addSbtPlugin("com.lightbend.sbt" % "sbt-java-formatter" % "0.4.4")
+addSbtPlugin("com.lightbend.sbt" % "sbt-java-formatter" % "0.7.0")
```

Comment:
This update include `javafmtCheckAll`. 

Previously we were using a script to format the files and check if there were any git diff.